lib/hotdog/commands.rb

```diff
@@ -262,37 +256,29 @@ def update_db(options={})
 
       def create_db(db, options={})
         options = @options.merge(options)
-        requests = {all_downtimes: "/api/v1/downtime", all_tags: "/api/v1/tags/hosts"}
         begin
-          parallelism = Parallel.processor_count
-          # generate payload before forking threads to avoid fetching keys multiple times
-          query = URI.encode_www_form(api_key: application.api_key, application_key: application.application_key)
-          responses = Hash[Parallel.map(requests, in_threads: parallelism) { |name, request_path|
-            [name, datadog_get(request_path, query)]
-          }]
+          all_tags = @source_provider.get_all_tags()
```

Comment:
This cannot be parallelized since `@source_provider` isn't thread-safe. Though the cost of serialized 2 HTTP calls are not that high actually. Just doing serial must be sufficient.